scripts/manage.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
This feels very Rails-like. The Django app should live in its own folder I think, I'm not familiar with this particular organization pattern in Python/Django. Maybe it would be best to place it in its own folder instead of the root of the repo? We will be storing a bunch more things in here as well, so we should organize it early.

For example, we can use the jungle book for inspiration (since we're all about elephants here!): https://lionking.fandom.com/wiki/Simba

Ideally, in the Python world, we want our packages to be as uniquely named as possible, so things like `control` or especially `www` in the global namespace make me think twice.